website/content/docs/oss/configuration/worker/pki-worker.mdx

```diff
@@ -0,0 +1,56 @@
+---
+layout: docs
+page_title: PKI Worker Configuration
+description: |-
+  PKI worker-specific parameters.
+---
+
+
+## PKI Worker Configuration
+PKI Workers authenticate to Boundary using a new certificate-based method, allowing for worker deployment
+without using a shared KMS. PKI Workers can also be configured using CLI commands.
+
+PKI Workers require an accessible directory, defined by `auth_storage_path`, for credential storage.
+Example (not safe for production!):
+
+```hcl
+worker {
+  auth_storage_path="/tmp/boundary/demo-worker-1"
+  initial_upstreams = ["10.0.0.1"]
+}
+```
+
+__Note__: `name` and `description` fields are not valid config fields for PKI workers. These fields are only valid for
+[KMS Workers][]. `name` and `description` can only be set for PKI workers through the API.
+
+# Complete Configuration Example
+
+```hcl
+listener "tcp" {
+	purpose = "proxy"
+	tls_disable = true
+	address = "127.0.0.1"
+}
+
+worker {
+  # Path for worker storage. Must be unique across workers
+  auth_storage_path="/tmp/boundary/demo-worker-1"
+
+  # Workers must be able to reach upstreams on :9201
```

Comment:
This is not necessarily true and I'd hate for someone to configure firewalls incorrectly by relying on this.

Reply:
does this read better?

`# Workers typically need to reach upstreams on :9201`

Reply:
added